benchmarks/scripts/plot_avg_benchmarks_over_time.py

```diff
@@ -116,10 +128,11 @@ def extract_compiler_versions(header):
 
         # Check if the version has changed
         if current_version != last_version_seen[compiler]:
-            text = f"{compiler}={current_version}"
+            text = f"{current_version}"
             xy = (row["date"], compiled_ratio)
             color = color_map[compiler]
-
+            renderer = fig.canvas.get_renderer()
+            previous_bboxes.append(ax[0].get_tightbbox(renderer))
```

Comment:
Can you share more on what this line is doing (adding the current tightbbox)? Is there a need to do it in the average compile time plot too?

Reply:
It was a failed experiment to get the labels to render more nicely. I've removed it.